### DISCUSSION TITLE
LibMambaUnsatisfiableError: Encountered problems while solving:

### DISCUSSION THREAD

**Discussion by yangdianxi** [idaholab/moose#29564] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

conda create -n moose moose-dev=2024.12.02=mpich
after：
Channels:
 - https://conda.software.inl.gov/public
 - conda-forge
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides __glibc >=2.28,<3.0.a0 needed by c-ares-1.34.3-heb4867d_0

Could not solve for environment specs
The following package could not be installed
└─ moose-dev ==2024.12.2 mpich is not installable because it requires
   └─ moose-libmesh ==2024.12.2 mpich_0, which requires
      └─ moose-petsc ==3.22.1.193.0g72c1e49ee3d mpich_0, which requires
         └─ moose-mpi =2024.11.13 *, which requires
            └─ moose-mpi-base [==2024.11.13 mpich|==2024.11.13 openmpi], which requires
               └─ c-ares ==1.34.3 heb4867d_0, which requires
                  └─ __glibc >=2.28,<3.0.0a0 *, which is missing on the system.

donot find resolve approach

**Comment by GiudGiud** [comment]
Hello

Can you please paste the output of the diagnostics script in moose/scripts?

**Reply by yangdianxi** [reply]
yes，thank you，I will send you the screenshot of the wrong picture
![Uploading 1.png…]()

**Reply by GiudGiud** [reply]
It looks like the upload did not work. 
If you can copy paste instead of screenshot, it's better for searching

**Comment by yangdianxi** [comment]
```
(base) user@user-virtual-machine:~$ conda create -n moose moose-dev=2024.12.02=mpich
Channels:
 - https://conda.software.inl.gov/public
 - conda-forge
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - nothing provides __glibc >=2.28,<3.0.a0 needed by c-ares-1.34.3-heb4867d_0

Could not solve for environment specs
The following package could not be installed
└─  moose-dev ==2024.12.2 mpich is not installable because it requires
   └─ moose-libmesh ==2024.12.2 mpich_0, which requires
      └─ moose-petsc ==3.22.1.193.0g72c1e49ee3d mpich_0, which requires
         └─ moose-mpi =2024.11.13 *, which requires
```

I'm sorry. I don't know why it didn't go up. This is the content of the error, I have pasted it all down, hope it helps
```
            └─ moose-mpi-base [==2024.11.13 mpich|==2024.11.13 openmpi], which requires
               └─ c-ares ==1.34.3 heb4867d_0, which requires
                  └─ __glibc >=2.28,<3.0.0a0 *, which is missing on the system.

(base) user@user-virtual-machine:~$ 
```

**Reply by GiudGiud** [reply]
what version of linux are you running? Seems to not be supported by our conda build

**Reply by milljm** [reply]
We might have had a bad set of packages that round. When I look into this a bit, it seems newly created packages (which are in the works) do not exhibit this issue.

Can you try the following?

```bash
conda create -n moose moose-dev mpich --channel https://conda.software.inl.gov/moose/pr-29104
```

Ref PR that fixes this issue: https://github.com/idaholab/moose/pull/29104

**Reply by yangdianxi** [reply]
I am using Wmware Ubuntu system.

**Reply by yangdianxi** [reply]
(base) user@user-virtual-machine:~$ conda create -n moose moose-dev mpich --channel https://conda.software.inl.gov/moose/pr-29104
Retrieving notices: done
Channels:
 - https://conda.software.inl.gov/moose/pr-29104
 - https://conda.software.inl.gov/public
 - conda-forge
Platform: linux-64
Collecting package metadata (repodata.json): failed

UnavailableInvalidChannel: HTTP 403 FORBIDDEN for channel moose/pr-29104 <https://conda.software.inl.gov/moose/pr-29104>

The channel is not accessible or is invalid.

You will need to adjust your conda configuration to proceed.
Use `conda config --show channels` to view your configuration's current state,
and use `conda config --show-sources` to view config file locations.

Oh,still report an error,that does not seem like a good idea.Dependent libraries do not match?.I do not  know what to do.

**Reply by milljm** [reply]
I am not sure why you can't peruse to that location... can you click on the link? 

https://conda.software.inl.gov/moose/pr-29104

If you can navigate there using your web browser, but not when using terminal:

```bash
$ curl https://conda.software.inl.gov/moose/pr-29104/index.html
<trimmed>


<td>X</td><td>X</td><td></td>      <td class="summary">The Workbench Analysis Sequence Processor (WASP) was developed to...</td>
    </tr>  </table>
  <address>Updated: 2024-12-18 20:16:05 +0000 - Files: 14</address>
</body>
</html>%  
```
Then this will indicate you are using some sort of Government/Corporate proxy preventing you access.

**Reply by yangdianxi** [reply]
WASP does not seem to be available.
Make sure to either run scripts/update_and_rebuild_wasp.sh in your MOOSE directory,
or set WASP_DIR to a valid WASP install



I think I have installed it, the link is ready to use, the installation package is ready to download, but there are some problems when testing again, I don't know what this WASP means?The above error occurred
